app/assets/javascripts/home.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
Unused asset and helper files like this can safely be removed from the project and added later should you want them.  Adding the following to `config/application.rb` _should_ prevent these files from being generated in the future, but that's dependent upon Rails version so YMMV:

```ruby
config.generators do |g|
  g.assets false
  g.helper false
end
```